src/resources/lib/modules/connman.py

```diff
@@ -806,7 +806,12 @@ def menu_connections(self, focusItem, services={}, removed={}, force=False):
                         if properties[prop]['type'] == 2:
                             result = unicode(result)
                         if properties[prop]['type'] == 3:
-                            result = unicode(len(result))
+                            if ('psk' or 'wep' or 'ieee8021x') in result:
```

Comment:
This test isn't working as intended - it will only return true for `psk`:
```
result=['ieee8021x','wep']
if ('psk' or 'wep' or 'ieee8021x') in result:
  print("yes")
else:
  print("no")
```
will print `no`